docs/articles/Processing/Activities.md

```diff
@@ -3,11 +3,11 @@ uid: Activities
 ---
 # Activities
 
-An `Activity` is the smallest separately executable step of a Process. Activities are defined as classes derived from `Activity<TParam>` and are always specific to a certain task or application. Activities are instantiated by dedicated modules in their respective domains and then executed by resources. Each activity has a set of required `capabilities` while the production resources each have a set of provided capabilities. For each activity the ProcessController searches for a resource with matching capabilities. If there is more than one matching then the first one found will be used. If the selected resource is [ReadyToWork](xref:Moryx.AbstractionLayer.Resources.ReadyToWork), the ProcessController sends the activity with a [StartActivity](xref:Moryx.AbstractionLayer.Resources.StartActivity) message to the resource to be executed there. All activities are stored permanently with the process for tracing purposes. An activity inside MORYX is represented by an obejct implementing [IActivity](xref:Moryx.AbstractionLayer.IActivity) and is usually derived from [Activity](xref:Moryx.AbstractionLayer.Activity).
+An [Activity](xref:Moryx.AbstractionLayer.IActivity) is the smallest separately executable step of a [Process](Processes.md). Activities are defined as classes derived from [Activity](xref:Moryx.AbstractionLayer.Activity) and are always specific to a certain task or application. Activities are instantiated by dedicated modules in their respective domains and then executed by resources. Each activity has a set of required [Capabilities](Capabilities.md) while the [production Resources](../Resources/Overview.md) each have a set of provided capabilities. For each activity the ProcessController searches for a resource with matching capabilities. If there is more than one matching then the first one found will be used. If the selected resource is [ReadyToWork](xref:Moryx.AbstractionLayer.Resources.ReadyToWork), the ProcessController sends the activity with a [StartActivity](xref:Moryx.AbstractionLayer.Resources.StartActivity) message to the resource to be executed there. All activities are stored permanently with the process for tracing purposes.
```

Comment:
Thanks for taking a look at this. Apparently it is even more outdated, since the execution session (ReadyToWork, ActivityStart, ...) were moved to the ControlSystem a while ago.

Reply:
Ok, so i guess deleting that part would work in this case, wouldn't it?

Reply:
And I think that considering that ReadyToWork is not part of the Abstraction layer anymore the [Constrains documentation](https://github.com/PHOENIXCONTACT/MORYX-AbstractionLayer/blob/dev/docs/articles/Processing/Constraints.md) should be checked as well

Reply:
Yes, starting from *"For each activity the ProcessControlle"* it can be deleted.

Reply:
>the Constrains documentation should be checked as well

Good catch, yes the second half is definitely outdated.